### ISSUE TITLE
need of new software's keys

### ISSUE THREAD

**Issue by im-adhavan1410** [qzenlebg/crypto-wallet-cracker#1]
I need u to crack for this software and pls add password with it bro 
https://github.com/BlockchaineCrack/crypto_cracker
when u see this request pls reply me

**Comment by qzenlebg** [comment]
Sorry but I didn’t understand your request could you give me more info

Le ven. 24 mai 2024 à 19:26, Adhavan1410 ***@***.***> a
écrit :

**Comment by im-adhavan1410** [comment]
this software is same like the software u r cracked that why I need u crack this file and wanted the key as a request as same u posted now

**Comment by qzenlebg** [comment]
It is exactly the same software so I don't understand why you want me to crack the other one but if you ask so I could provide it to you

**Comment by almightynan** [comment]
To start the program it requires you to enter an activation key, see attachment
![image](https://github.com/qzenlebg/crypto-wallet-cracker/assets/93263738/2f0c7c17-3c0e-4829-be30-0432a99d11c8)

**Comment by im-adhavan1410** [comment]
Yeah I know bro but in his repository one he cracked the another program
(Crypto Wallet Cracker) that's the reason I want him to crack this software
(Bruto Force Seed). Will he able to help me with that..! This software is 10X faster than this previous cracked program bro..

**Comment by im-adhavan1410** [comment]
> It is exactly the same software so I don't understand why you want me to crack the other one but if you ask so I could provide it to you 

Yeahh bro I need that software with password Can u provide it for me

**Comment by im-adhavan1410** [comment]
> It is exactly the same software so I don't understand why you want me to crack the other one but if you ask so I could provide it to you 

hey bro I found an another software which the wallet is found faster than other software but the problem is I don't have key to export the wallet 


when u are free reply to this message

**Comment by im-adhavan1410** [comment]
> It is exactly the same software so I don't understand why you want me to crack the other one but if you ask so I could provide it to you 

hey bro did u crack it

**Comment by im-adhavan1410** [comment]
> It is exactly the same software so I don't understand why you want me to crack the other one but if you ask so I could provide it to you 

hey bro did u crack this software

**Comment by salencobaba** [comment]
Can I know different between (Bruto Force Seed). and the (Crypto Wallet Cracker)? because to me  they have different interface but same workings

**Comment by almightynan** [comment]
brute force seed focuses on systematically guessing a wallet's recovery seed phrase to crack the wallet targeting the similar word based phrases. wallet cracker is a broader tool that targets various wallet security aspects, including seed phrases, private keys, passwords and encrypted files

**Comment by salencobaba** [comment]
This is scam! I dont trust this

**Comment by almightynan** [comment]
@salencobaba i never mentioned this one saying it is *not a scam*. most crypto wallet crackers are scams now they just out here trying to steal your $$ or info, promising stuff they can’t do. ain’t no legit way they cracking wallets like that. you have to be careful and not fall for the bait or just find a real job.

**Comment by salencobaba** [comment]
Oh sorry I didt get that well before I think you are promoting them!!
Many Thanks

**Comment by almightynan** [comment]
nah, not promoting them at all! just saying most of them are fake and tryna scam ppl.

**Comment by salencobaba** [comment]
Yes all of them are not real no real ones at all!
